config/packages/serializer/Centreon/MonitoringServer.MonitoringServer.yml

```diff
@@ -5,7 +5,10 @@ Centreon\Domain\MonitoringServer\MonitoringServer:
             groups: [monitoringserver_main, monitoringserver_min]
         name:
             type: string
-            groups: [monitoringserver_main, monitoringserver_min]
+            groups:
+              - monitoringserver_main
+              - monitoringserver_min
+              - mapper_monitoringserver
```

Comment:
Can you update the serialization name ? Because `mapper` has no meaning in centreon repository

Reply:
Indeed!